docs/AsyncWrap/README.md

```diff
@@ -153,13 +153,13 @@ asyncWrap.disable();
 #### The Hooks
 
 Currently there are 4 hooks: `init`, `pre`, `post` `destroy`. The `this`
-variable refers to the handle object. The `init` hook has three extra arguments
-`provider`, `uid` and `parent`. The `destroy` hook also has the `uid` argument.
+variable refers to the handle object, they all have a `uid` argument, finally
+`init` provides extra information about the creation of the handle object.
```

Comment:
I think specifying all details is overdoing it, the signature clearly shows the details and the rest of the section is dedicated to explaining these arguments. This provides the general thought behind those extra arguments, which I actually find much more useful than just writing out their names.